app/controllers/redirect_controller.rb

```diff
@@ -1,45 +0,0 @@
-class RedirectController < ActionController::Base
```

Comment:
I'm not sure we want to take this out actually. Infact this will break group subdomains wont it?